### ISSUE TITLE
为知笔记又挂了

### ISSUE THREAD

**Issue by CharMingF** [suziwen/markdownxiaoshujiang#459]
更新的V2版本能用了几天，这几天又出问题了，可以看到文章列表，但是点击文章和保存都失败了。

**Comment by CharMingF** [comment]
另外，好像，dropbox也不能用了，登录进去说，授权的时候 没有权限。

**Comment by suziwen** [comment]
估计 wiz 又调整了 api
dropbox 我这边测试了下绑定授权，没有你说的情况。

**Comment by CharMingF** [comment]
![snipaste20170716_081944](https://user-images.githubusercontent.com/7530342/28243517-aceb53b2-69ff-11e7-8ad3-5fd05393b951.jpg)
我用谷歌的登录的dropbox的，登录进去就这样了。

**Comment by CharMingF** [comment]
@suziwen Production approval

When you first create a Dropbox API app, it's given development status. Your app functions the same as any production status app except that it can only be linked with up to 500 total Dropbox users. However, once your app links 50 Dropbox users, you will have two weeks to apply for and receive production status approval before your app's ability to link additional Dropbox users will be frozen, regardless of how many users between 0 and 500 your app has linked. For more details, please read the Applying for production status section below.
You can enable development status from your app's info page on the App Console by clicking Enable additional users. Many apps such as those for demos, hackathons, staging environments, and internal tools can stay in development status. However, if you'd like to open your app to more users, you'll need to apply for production status.
Applying for production status
If you want to share your app with the world, apply for production status from your app's info page, accessible via the App Console. Before applying for production, make sure your app adheres to the developer branding guidelines and terms and conditions. It will be rejected if it doesn't.
When you apply for production status, you'll be prompted to supply additional information such as how your app uses the API, and to upload an icon for the app. Your request will then be submitted to us for review.
While you may submit your request for production status anytime, it will not be reviewed until your app has linked with at least 50 Dropbox users, indicating that it will likely require production status.
Once your app has linked 50 Dropbox users, you will have two weeks to apply for and receive production status approval, if you haven't already done so. If after two weeks you haven't applied for and received production approval, your app's ability to link additional Dropbox users will be frozen until you submit an application and are approved for production status. Once your production status application has been submitted, your app will be reviewed to ensure it adheres to the above guidelines.
Please note, for apps that have been temporarily frozen, the only way to unfreeze your app is to apply for and receive production status. Simply unlinking all of your users will not unfreeze your app.
When submitting an application, please be descriptive! The more info you provide when applying, the faster we may be able to approve your app once it reaches 50 linked Dropbox users.
Once your app is approved for production status, any number of Dropbox users can link to your app. Keep in mind that once your app has production status, you can't change its name. Make sure you're comfortable with your app's configuration before applying. Remember, your app is required to adhere to the production status requirements even after it has been approved.
Note: As part of the deprecation of API v1, Dropbox will no longer approve for production apps that continue to query API v1 endpoints. Please ensure that your app is only querying API v2 endpoints. Read more about the deprecation of API v1 here.
Exceptions
Some apps may be eligible for production status review before reaching 50 linked Dropbox users. If you feel there's a compelling reason for why your app should be reviewed ahead of time, please provide evidence and a detailed explanation under the Request early review field in the production status application form. Otherwise, your app will be reviewed soon after linking 50 Dropbox users.

**Comment by suziwen** [comment]
window 版本的话，可以自己先替换下这个文件 `安装目录/plugins/plugins/wizv2/wiz.js`
[wiz.js.zip](https://github.com/suziwen/markdownxiaoshujiang/files/1243937/wiz.js.zip)

**Comment by suziwen** [comment]
dropbox 好像是我当时没申请成正式的 :frowning_face: .
不过刚看了下 dropbox 的通知，好像他们要禁止使用旧 api, 全部转换成新 api 接口。先标记下。

**Comment by suziwen** [comment]
最新版本 5.0.0 已经修复发布
